api/v1alpha1/awsvalidator_types.go

```diff
@@ -40,6 +40,19 @@ type AwsAuth struct {
 	SecretName string `json:"secretName,omitempty" yaml:"secretName,omitempty"`
 	// Option 2: specify a service account (EKS)
 	ServiceAccountName string `json:"serviceAccountName,omitempty" yaml:"serviceAccountName,omitempty"`
+	// Option 3: define STS authentication properties
```

Comment:
Suggestion: `// STS authentication properties (optional, in conjunction with Option 1 or 2)`